Formula/waffle.rb

```diff
@@ -29,6 +29,10 @@ class Waffle < Formula
   end
 
   def install
+    # Fix: Library/Developer/CommandLineTools/SDKs/MacOSX13.sdk/usr/include/netinet/ip.h:102:2:
+    # error: unknown type name 'u_char'; did you mean 'char'?
+    ENV.append_to_cflags "-D_DARWIN_C_SOURCE" if OS.mac?
```

Comment:
Can we upstream this? I'm a bit wary of this having unintended consequences.

Reply:
I have opened an upstream issue and added it to the comment: https://gitlab.freedesktop.org/mesa/waffle/-/issues/115